backend/src/backend/pdf/validator.py

```
@@ -0,0 +1,44 @@
+import os
+from typing import Tuple, Optional
+import pdfplumber
+from src.backend.app.logger import logger, log_traceback
```

Ruff (0.8.0)

`src.backend.app.logger.logger` imported but unused

Remove unused import: `src.backend.app.logger.logger`

(F401)